src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -0,0 +1,284 @@
+        /// Gets or sets current VT100 escape sequence for header 6.
+        /// </summary>
+        public string Header6 { get; set; }
+
+        /// <summary>
+        /// Gets or sets current VT100 escape sequence for code inline and code blocks.
+        /// </summary>
+        public string Code { get; set; }
+
+        /// <summary>
+        /// Gets or sets current VT100 escape sequence for links.
+        /// </summary>
+        public string Link { get; set; }
+
+        /// <summary>
+        /// Gets or sets current VT100 escape sequence for images.
+        /// </summary>
+        public string Image { get; set; }
+
+        /// <summary>
+        /// Gets or sets current VT100 escape sequence for bold text.
+        /// </summary>
+        public string EmphasisBold { get; set; }
+
+        /// <summary>
+        /// Gets or sets current VT100 escape sequence for italics text.
+        /// </summary>
+        public string EmphasisItalics { get; set; }
+
+        /// <summary>
+        /// Get the property as an rendered escape sequence.
+        /// This is used for typesps1xml for displaying.
+        /// </summary>
+        /// <param name="propertyName">Name of the property to get as escape sequence.</param>
+        /// <returns>Specified property name as escape sequence.</returns>
+        public string AsEscapeSequence(string propertyName)
+        {
+            var propertyValue = this.GetType().GetProperty(propertyName)?.GetValue(this) as string;
+
+            if (!string.IsNullOrEmpty(propertyValue))
+            {
+                return string.Concat(Esc, propertyValue, propertyValue, Esc, "[0m");
+            }
+            else
+            {
+                throw new InvalidOperationException();
+            }
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="MarkdownOptionInfo"/> class and sets dark as the default theme.
+        /// </summary>
+        public MarkdownOptionInfo()
+        {
+            SetDarkTheme();
+        }
+
+        /// <summary>
+        /// Set all preference for dark theme.
+        /// </summary>
+        public void SetDarkTheme()
+        {
+            Header1 = "[7m";
+            Header2 = "[4;93m";
+            Header3 = "[4;94m";
+            Header4 = "[4;95m";
+            Header5 = "[4;96m";
+            Header6 = "[4;97m";
+            Code = "[48;2;155;155;155;38;2;30;30;30m";
+            Link = "[4;38;5;117m";
+            Image = "[33m";
+            EmphasisBold = "[1m";
+            EmphasisItalics = "[36m";
+        }
+
+        /// <summary>
+        /// Set all preference for light theme.
+        /// </summary>
+        public void SetLightTheme()
+        {
+            Header1 = "[7m";
+            Header2 = "[4;33m";
+            Header3 = "[4;34m";
+            Header4 = "[4;35m";
+            Header5 = "[4;36m";
+            Header6 = "[4;30m";
+            Code = "[48;2;155;155;155;38;2;30;30;30m";
```

Comment:
since you're reusing a bunch of these, I'd be happier if they were constants

Reply:
Fixed.